acceptance/acceptance_test.go

```diff
@@ -718,3 +727,84 @@ func filterHeaders(h http.Header, includedHeaders []string) http.Header {
 func isTruePtr(value *bool) bool {
 	return value != nil && *value
 }
+
+// runWithTail starts cmd.Run() and concurrently tails the file.
+// For each new line that appears in the file, it calls t.Log(line).
+// When cmd.Run() finishes, it uses os.Stat(filename) to get the file’s final size,
+// and stops tailing once the file offset is greater or equal to that size.
+// The function returns the error from cmd.Run().
+func runWithTail(t *testing.T, cmd *exec.Cmd, filename string) error {
```

Comment:
Can't we just use `io.Pipe`, plug it in into stdout of cmd.Run and log from there? Wouldn't it be easier?

Reply:
It would be a valid approach as well but I'm not sure it actually makes a big difference in complexity.

I also like that here that we write to output.txt in the most efficient way and use the same approach whether -tail is set or not.

Reply:
Updated to use io.Pipe - https://github.com/databricks/cli/pull/2467/commits/d1f1f190426b299b46e1e8231c2694cdd0cf9453

You're right, it's a bit simpler.